src/App.js

```diff
@@ -35,9 +37,11 @@ function App() {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
   const [city, setCity] = useState("New York City");
+  const [countryCode,setCountryCode] = useState("US");
   const [results, setResults] = useState(null);
   const [objects, setObjects] = useState([]);
   const [content, setcontent] = useState("");
+  const { data, setData } = useFetch();
```

Comment:
I think we should have a more descriptive name for these since it's not obvious what the data is

Reply:
I have made all the above mentioned improvements now and solved the merge conflicts